### PR TITLE
Add .eslintrc to truffledb

### PR DESCRIPTION
**Pull request by tenthirtyone**
Ran into this with @cds-amal this week while working on unit tests. Husky will bark at you after making changes to existing tests without this change.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -2,7 +2,9 @@
   "root": true,
   "env": {
     "es2021": true,
-    "node": true
+    "node": true,
+    "mocha": true,
+    "jest": true
```
Comment (gnidan) [inline .eslintrc.json:7]: you can have both of these???

**Review by haltman-at — COMMENTED** [review]
I don't think this is the right way to do this, because this will affect everything.  Or, at the least, it's not the way we're already handling this currently!  This is what `.eslintc.package.json` is for; the way we currently handle this problem is to put, in packages that need it, a `.eslintrc.json` that imports the global `.eslintrc.package.json`.  You can see this in `box`, in `debugger`, etc.  So, whatever package you need this for, just add the same thing there!

...or, if you really think this is the right way to do it, then this PR should in addition delete `.eslintrc.package.json` and all the files that reference it. :P

**Comment by tenthirtyone** [comment]
@haltman-at no strong feelings at all. Your ears must have been burning @cds-amal and I were on a call talking about doing the exact same thing with `tsconfig.json`. I did not know it was already setup for eslint.

I'll make the change and send it up.

File: .eslintrc.json (hunk 1 above)
Comment (cds-amal) [reply .eslintrc.json]: From the [docs](https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments)

> These environments are not mutually exclusive, so you can define more than one at a time.

**Review by haltman-at — APPROVED** [review]
Thanks for getting this!

**Pull request merged**